### PR TITLE
@mzikherman - adds artists to follow rail on homepage (lab feature)

### PR DESCRIPTION
**Pull request by broskoski**
artists-to-follow-rail

### CODE REVIEW

File: components/follow_button/view.coffee [hunk 1]
```diff
@@ -12,7 +12,14 @@ module.exports = class FollowButton extends Backbone.View
     'touchstart': () -> @$el.removeClass "no-touch"
 
   initialize: (options) ->
-    { @following, @notes, @modelName, @href, @context_page, @context_module } = options
```
Comment (mzikherman) [inline components/follow_button/view.coffee]: what happened to `@context_module`?

Comment (broskoski) [inline components/follow_button/view.coffee]: Ah good catch!

**Comment by broskoski** [comment]
👋

**Comment by mzikherman** [comment]
LGTM! Curious why the lab feature lockdown, and not just a general release?

**Comment by broskoski** [comment]
Just for QA purposes!

**Comment by mzikherman** [comment]
Aight, cool!

**Pull request merged**